### ISSUE TITLE
Implement and maintain a PPA for Barrier older/new releases

### ISSUE THREAD

**Issue by shymega** [debauchee/barrier#826]
As well as providing Snaps and Flatpaks for Barrier, it would be
beneficial to provide a PPA for Barrier. This would allow for older
versions of Ubuntu (or derivatives) to use the latest version of
Barrier. Although a Snap/Flatpak _is_ available, some users may wish
to use Barrier without tht.

I will look into implementing this soon, this issue is merely a
tracker.

**Comment by fcastilloec** [comment]
I totally agree with this. I just installed the `snap` version and I couldn't make it work. The server wasn't seeing anybody... Had to install my distro version, which is old, and then it worked.

**Comment by shymega** [comment]
Hi,

I am no longer involved in the Barrier project, and have moved to a fork, whilst also beginning work on my new primary focus: a hybrid KVM. As I am no longer involved with Barrier, I cannot assist anymore, as I have no access to the administration/management tools.

Apologies for the inconvenience.

**Comment by shymega** [comment]
You reacted with the confused emoji, I'm just explaining there's nothing I can do with this issue now.
